RecoParticleFlow/PFClusterProducer/plugins/SimMappers/ComputeClusterTime.h

```diff
@@ -0,0 +1,133 @@
+#ifndef ComputeClusterTime_h
```

Comment:
Although it is unlikely that there are two classes with the same name in CMSSW,  include guards should better include the full path or alike, e.g. ```RecoParticleFlow_PFClusterProducer_ComputeClusterTime_h```